sklearn/neighbors/base.py

```diff
@@ -345,8 +355,9 @@ def kneighbors_graph(self, X, n_neighbors=None,
 
         Parameters
         ----------
-        X : array-like, shape = [n_samples, n_features]
-            Sample data
+        X : array-like, shape (n_query, n_features), \
```

Comment:
I was hoping to get away with resolving this along with others like it in future work! Will that fly by you?